### DISCUSSION TITLE
Authentication via Keycloak

### DISCUSSION THREAD

**Discussion by ngocnn97** [DefectDojo/django-DefectDojo#10382] [Q&A]
Hi, 

i am trying to connect defect dojo and keycloak
Did exactly everything in this document: [document](https://defectdojo.github.io/django-DefectDojo/integrations/social-authentication/#configure-keycloak)

But I get error:
![image](https://github.com/DefectDojo/django-DefectDojo/assets/55728023/aa537a28-ae46-46cb-8806-5028c3a23e4e)

Anyone can help me !

**Comment by mtesauro** [comment]
You may have a larger audience if you ask this in OWASP's Slack instead of here (GH Discussions):

Info on joining OWASP's Slack and the `#defectdojo` channel is at https://github.com/DefectDojo/django-DefectDojo/tree/master?tab=readme-ov-file#community-getting-involved-and-updates

**Reply by ngocnn97** [reply]
> You may have a larger audience if you ask this in OWASP's Slack instead of here (GH Discussions):
> 
> Info on joining OWASP's Slack and the `#defectdojo` channel is at https://github.com/DefectDojo/django-DefectDojo/tree/master?tab=readme-ov-file#community-getting-involved-and-updates

i asked on Slack but no one answered me, can you help me @mtesauro

**Reply by mtesauro** [reply]
Sorry, beyond knowing the keycloak is a SSO/IdP, I have no real experience with it nor the code that works with it in DefectDojo. I've not spent any time with the various identity providers in the code base.

**Reply by ngocnn97** [reply]
Maybe I should be patient and wait for someone to help here :))))
